### ISSUE TITLE
sudo not working after upgrading to Debian 10 testing (Buster)

### ISSUE THREAD

**Issue by DeicPro** [CypherpunkArmory/UserLAnd#80]
I upgraded the system modifying the `/etc/apt/sources.list` file replacing:
```
deb http://deb.debian.org/debian/ stable main contrib non-free
#deb-src http://deb.debian.org/debian/ stable main contrib non-free
deb http://deb.debian.org/debian/ stable-updates main contrib non-free
#deb-src http://deb.debian.org/debian/ stable-updates main contrib non-free
```
With:
```
deb http://deb.debian.org/debian/ testing main contrib non-free
#deb-src http://deb.debian.org/debian/ testing main contrib non-free
deb http://deb.debian.org/debian/ testing-updates main contrib non-free
#deb-src http://deb.debian.org/debian/ testing-updates main contrib non-free
```
Then I did `sudo apt update` and `sudo apt upgrade`. After all got installed, `sudo` not working anymore showing:
```
sudo: error in /etc/sudo.conf, line 0 while loading plugin "sudoers_policy"                       
sudo: unable to load /usr/lib/sudo/sudoers.so: /usr/lib/sudo/sudoers.so: cannot stat shared object
: Bad file descriptor                            
sudo: fatal error, unable to load plugins
```

**Comment by corbinlc** [comment]
I have this fixed locally.  Will roll out as mentioned with the first production release.

**Comment by Root-voidX** [comment]
Hey @corbinlc we look foward to test the new beta release, but is it possible that linux kernel versions might be an issue too? Have you guys run into anything while using sd845 devices?

**Comment by corbinlc** [comment]
This should be fixed with release 0.2.3.  #139 
Please file separate issues for anything new.

**Comment by qqqwwweee123-ping** [comment]
update to rootfs v0.0.4 debian 10 buster?
